receiver/hostmetricsreceiver/internal/scraper/processscraper/process_scraper_test.go

```diff
@@ -133,6 +133,7 @@ func assertProcessResourceAttributesExist(t *testing.T, resourceMetrics pmetric.
 		internal.AssertContainsAttribute(t, attr, conventions.AttributeProcessCommand)
 		internal.AssertContainsAttribute(t, attr, conventions.AttributeProcessCommandLine)
 		internal.AssertContainsAttribute(t, attr, conventions.AttributeProcessOwner)
+		internal.AssertContainsAttribute(t, attr, "process.parent_pid")
```

Comment:
Did you submit a PR to semantic conventions?

Reply:
no, I thought you wanted me to wait for this PR first.  I'll do that now.